### ISSUE TITLE
`web-tree-sitter` not working with Node 18.1+ 

### ISSUE THREAD

**Issue by Krzysztof-Cieslak** [tree-sitter/tree-sitter#1765]
As in title - basic Node.js script using `web-tree-sitter` is working flawlessly on the node 18.0 (and earlier) while it fails on 18.1 (and later)

Reproduction in this repository: https://github.com/Krzysztof-Cieslak/TreeSitterTest

<img width="1421" alt="image" src="https://user-images.githubusercontent.com/5427083/171426312-b5f8ddad-48d4-48e7-be01-06cc36717b77.png">

**Comment by razzeee** [comment]
So this means, that you will need to do a release, as that file is not checked in?

**Comment by grahambates** [comment]
Unfortunately building with the lastest version of emscripten results in this issue https://github.com/tree-sitter/tree-sitter/issues/1788.

**Comment by sguilliaci** [comment]
The two bindings that allow to use tree-sitter with Javascript are broken with node 18:
- node-tree-sitter (https://github.com/tree-sitter/node-tree-sitter/issues/106)
- web-tree-sitter (this issue)

**Comment by sguilliaci** [comment]
As a workaround, just at this at the very beginning of your app:
```ts
try {
	//@ts-ignore
	delete WebAssembly.instantiateStreaming
} catch {}
```

Works like a charm

**Comment by tpope** [comment]
I can confirm that this was fixed by #1913 and can be closed. Eagerly awaiting a stable release containing the fix. :pray:

**Comment by bmuskalla** [comment]
Is there an ETA for when treesitter does the next release to consume this fix?
